Source/LinqToDB/Sql/IsNullableType.cs

```diff
@@ -14,6 +14,7 @@ public enum IsNullableType
 			SameAsSecondParameter,
 			SameAsThirdParameter,
 			SameAsLastParameter,
+			IfAllParametersNullable
```

Comment:
Reduce future code difference when new enum values are added.

```suggestion
			IfAllParametersNullable,
```

Reply:
added